ccswbs/gus
# Refactor breadcrumb component and other fixes

# Summary of changes
The `breadcrumbs.js` component has been refactored to show unlimited levels as opposed to just two. Additional changes have been implemented to fix glitches and accessibility warnings in other components.

## Frontend
- Add new functions to `breadcrumbs.js` to handle multi-level trails and remove unneeded code
- Remove unused variable from `events.js` and add new code to handle events with no external URL
- Tweak `navTabs.js` code to address jsx-a11y warning re non-interactive elements getting an interactive role (see https://getbootstrap.com/docs/5.0/components/navs-tabs/#javascript-behavior for more details)
- Remove unused variable from `heroVideo.js`

## Backend
None

# Test Plan
- Go to https://tqtest.gatsbyjs.io/studentexperience
- Choose `Off-Campus Living` from the `Advising & Support` dropdown
- Click the `Right Foot Forward` button
- Verify the breadcrumb trail shows [Rental Living](https://tqtest.gatsbyjs.io/studentexperience/rental-living) > [Advising & Support](https://tqtest.gatsbyjs.io/studentexperience/advising-and-support) > [Off-Campus Living](https://tqtest.gatsbyjs.io/studentexperience/off-campus-living) > [Rental Resources](https://tqtest.gatsbyjs.io/studentexperience/rental-resources) > Right Foot Forward
- Verify the breadcrumb links go to the correct pages
- Go to https://tqtest.gatsbyjs.io/gceb3 and repeat the above steps to verify the breadcrumb trail there (note that these breadcrumb links go to absolute URLs on the OVC site)
- Go to https://tqtest.gatsbyjs.io/choose-u-of-g
- Verify the link for the Mar 11 event `ONLINE: China - Its Rise and Consequences` goes to the correct link on the UoG news site
